### PR TITLE
Describe magnifier button

### PR DESCRIPTION
**Pull request by MarySammal**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md [hunk 1]
```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
 
 ![WinFormsPropertyGrid-ResetValue](../../../../../images/eurd-win-FormsPropertyGrid-ResetValue.png)
 
+## Set Color Properties
+
+You can use Magnifier to set color properties.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: Change "use Magnifier" to "use the Magnifier"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md [hunk 2]
```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
...
+
+![](../../../../../images/eurd-win-magnifier-button.png)
+
+Do the following to use Magnifier:
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: Change "use Magnifier" to "use the Magnifier"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md [hunk 3]
```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
...
+
+- Click the Magnifier button.
+- Move the invoked Magnifier on the screen to find the color you want to set.
+
+	![](../../../../../images/eurd-win-magnifier-get-color.png)
+
+	You can zoom out the magnified zone. To do this use one of the following options:
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: You can zoom out the magnified zone.
Can we say "You can use the Maginifier to zoom out."

Also change the second sentence to "Use one of the following options to do this:"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md [hunk 4]
```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
...
+	
+	- Hold Ctrl and press + / -
+	- Scroll the mouse wheel
+
+- Click to set the color property to the found color.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: What do you mean by "found color"? (This does not sound correct.)

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md [hunk 5]
```diff
@@ -44,6 +44,30 @@ Right-click a property's editor to reset the value.
...
+
+	![](../../../../../images/eurd-win-magnifier-set-color.png)
+
+Right click to cancel Magnifier mode.
```
Comment (albertov05) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: Change "cancel Magnifier mode" to "cancel the Magnifier mode"

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md (hunk 4 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: Changed to "Click to set the color property to the selected color."

Comment (albertov05) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/report-designer-tools/ui-panels/property-grid.md]: Thank you

**Pull request merged**